modules/json/json-plugin.c

```diff
@@ -42,6 +43,12 @@ static Plugin json_plugins[] =
     .name = "format_json",
     .construct = filterx_format_json_new_construct,
   },
+  {
+    .type = LL_CONTEXT_FILTERX_FUNC,
+    .name = "cached_json_file",
```

Comment:
I'd probably rename this, if it's not a big hassle. We have started to write functions a'la parse_kv() etc, so it should be `cache_json_file()` or something like that.

Similarly, if we were to write a simple json parsing function, that would be parse_json()

Reply:
Just pushed a separate patch to rename this, we can revert that as a whole if you don't agree.